downstream/titles/release-notes/async/aap-25-4-20-nov.adoc

```diff
@@ -0,0 +1,74 @@
+[[aap-25-4-20-nov]]
+
+= {PlatformNameShort} patch release November 20, 2024
+
+The following enhancements and bug fixes have been implemented in this release of {PlatformNameShort}.
+
+== Enhancements
+
+* With this release, a redirect page has now been implemented that will be exhibited when you navigate to the root ``/`` for each component's stand-alone URL. The API endpoint remains functional. This affects {EDAName}, {ControllerName}, {OperatorPlatformNameShort}, and {OCPShort}.
+
+
+== Bug fixes
+
+=== General
+
+With this update, the following CVEs have been addressed:
+
+link:https://access.redhat.com/security/cve/cve-2024-9902[CVE-2024-9902] ansible-core: Ansible-core user may read/write unauthorized content.
+
+link:https://access.redhat.com/security/cve/cve-2024-8775[CVE-2024-8775] ansible-core: Exposure of sensitive Information in Ansible vault files due to Improper logging.
+
+
+=== {PlatformNameShort}
+
+* Fixed an issue where the user is unable to filter out hosts on inventory groups where it returns a *Failed to load* options on {PlatformNameShort} UI.
```

Comment:
**is** -> **was**
**returns** -> **returned**
Past tense for fixed issues. This is a global comment.

Reply:
Done.